common/connection.go

```diff
@@ -488,6 +507,9 @@ func (c *Connection) sendLoop() {
 	}
 }
 
+// Close cleanly closes the WebSocket connection.
+// It returns an error if sending the Close control frame fails.
+// TODO: Remove this and close(), since nothing calls them anymore.
 func (c *Connection) Close(args ...goja.Value) {
```

Comment:
BTW, should we do the TODO here? We're not using this anymore, and I doubt that will change, so might as well remove it?

Reply:
`c.close()` is called from several places. On the other hand, if `Close()` is not being used anymore, why not call `Stop` -> `Close` instead? `connection.Stop` sounds weird.

Reply:
> `c.close()` is called from several places.

Ah, right, I missed the `closeCh` readers. Fair enough, we shouldn't delete it. And there's not much reason to remove `Close()` either then. It's only used in tests, but we can keep those.

I'll remove the TODO then.


> On the other hand, if `Close()` is not being used anymore, why not call `Stop` -> `Close` instead? `connection.Stop` sounds weird.

The rename would be confusing in that case, since `Stop()` doesn't actually close the connection, just... stops it (or rather, signals the `recvLoop()` error handling to ignore the `unexpected EOF` error). I couldn't think of a clearer name, but since we're not deleting `Close()`, I think we can keep it. Otherwise, I'm open to better name suggestions, just not `Close`.

Reply:
I don't think we're bikeshedding yet, just picking a better name. :)

Again, I don't disagree with you about `Stop`. Just that `Close` wouldn't be an improvement.

`IgnoreIOErrors()` is clearer, so I'll rename it to that. Thanks.

Reply:
Should be resolved by eb75063. I couldn't think of better names for the internal channel and unexported function to check it, so I settled on `closing` and `isClosing`.